src/protocol/resp/src/request/mod.rs

```diff
@@ -59,10 +62,15 @@ pub use hset::*;
 pub use hvals::*;
 pub use sadd::*;
 pub use set::*;
+// response codes for klog
+const MISS: u8 = 0;
+const HIT: u8 = 4;
+const STORED: u8 = 5;
+const NOT_STORED: u8 = 9;
```

Comment:
I don't think this is the right code to use for what we're using it for? `NOT_STORED` is the result of conditions for add/replace not being met. See: https://github.com/memcached/memcached/blob/master/doc/protocol.txt#L277

Reply:
@brayniac do you think preserving the memcached meanings in Klog would be helpful? I assumed there was a chance it would be better than defining an entirely new set of values just for RESP, mostly to make it simpler for endusers, but I hadn't really asked that question.

Is Klog something that someone would actually run in prod, anyways? My preference would be for Klog to render the code as a full string, but I could see how the performance of that would be impossible to justify. Right now we have like, something that's sort of a compact textual representation (i.e. not binpacked or anything but with minimal characters being logged). Curious if this would be considered a performance sensitive enough area of the codebase to preclude logging anything but a single character